src/bin/repo-sync-test/worker-routine.ts

```diff
@@ -28,10 +28,8 @@ let isRepoSynced: boolean = true
 export async function workerRoutine(config: WorkerConfig): Promise<void> {
   // Create sync clients
   const syncClients = Object.entries(config.clusters).reduce<SyncClient[]>(
-    (syncClients, [clusterName, urls]) => {
-      const clients = urls.map(
-        serverUrl => new SyncClient(serverUrl, clusterName)
-      )
+    (syncClients, [_, urls]) => {
```

Comment:
If `_` isn't used, wouldn't it be `Object.values`?

Reply:
Nice catch. I was too quick in my approach here. I'll fix it.